Cloud Application Manager/Deploying Anywhere/Using-CenturyLink-Private-Cloud-with-VMC-on-AWS.md

```diff
@@ -0,0 +1,143 @@
+{{{ "title": "Using the CenturyLink Private Cloud with VMC on AWS",
+"date": "12-05-2019",
+"author": "Sharon Wang",
+"keywords": ["cam", "cloud application manager", "vmware", "aws", "vmc", "cpc", "centurylink private cloud"],
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+**In this article:**
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Required Permissions](#required-permissions)
+* [Bootstrapping virtual machine templates with the Cloud Application Manager Agent](#bootstrapping-virtual-machine-templates-with-the-cloud-application-manager-agent)
+* [Registering a VMC provider in Cloud Application Manager](#registering-a-vmc-provider-in-cloud-application-manager)
+* [Registering Instances to Your VMC Provider](#registering-instances-to-your-vmc-provider)
+* [Deploying in Your VMC](#deploying-in-your-vmc)
+* [Getting General Support](#getting-general-support)
+
+### Overview
+
+This article is meant to assist users of Cloud Application Manager to configure a CenturyLink Private Cloud with VMC on AWS provider, then deploy or register a virtual machines with it.
+
+### Audience
+
+All users of Cloud Application Manager using CenturyLink Private Cloud with VMC on AWS providers.
```

Comment:
Please, use the official name `CenturyLink Private Cloud for VMware Cloud on AWS`
```suggestion
All users of Cloud Application Manager using CenturyLink Private Cloud for VMware Cloud on AWS (CPC for VMC) providers.
```